### PR TITLE
docs: add `Edit this page`

### PR DESCRIPTION
**Pull request by ghost**
Update theme to support `Edit this page`:
https://gitea.com/gitea/theme/pulls/116

Screenshot:
Edit this page

Signed-off-by: Xinyu Zhou <i@sourcehut.net>

### CODE REVIEW

**Review by lunny — APPROVED** [review]

File: docs/config.yaml [hunk 1]
```diff
@@ -23,6 +23,8 @@ params:
   goVersion: 1.19
   minNodeVersion: 14
   search: nav
+  github_doc_repository: "https://github.com/go-gitea/gitea"
+  github_doc_repository_path: "docs/content"
```
Comment (silverwind) [inline docs/config.yaml]: Can these be renamed to `repo`and `docContentPath`? I think it would be more fitting. camelCase to follow existing vars.

Comment (ghost) [reply docs/config.yaml]: Sure. please review https://gitea.com/gitea/theme/pulls/116

**Review by delvh — APPROVED** [review]

**Review by silverwind — APPROVED** [review]

**Comment by jolheiser** [comment]
Does this PR also require the partial? Docs build is failing in CI.

**Comment by zeripath** [comment]
yup it looks like something is missing:

```
Building sites … ERROR 2022/11/30 18:36:32 render of "page" failed: "/drone/src/docs/themes/gitea/layouts/doc/single.html:14:8": execute of template failed: template: doc/single.html:14:8: executing "doc/single.html" at <partial "edit_this_page" .>: error calling partial: partial "edit_this_page" not found
ERROR 2022/11/30 18:36:32 render of "page" failed: "/drone/src/docs/themes/gitea/layouts/doc/single.html:14:8": execute of template failed: template: doc/single.html:14:8: executing "doc/single.html" at <partial "edit_this_page" .>: error calling partial: partial "edit_this_page" not found
ERROR 2022/11/30 18:36:32 render of "page" failed: "/drone/src/docs/themes/gitea/layouts/page/single.html:14:8": execute of template failed: template: page/single.html:14:8: executing "page/single.html" at <partial "edit_this_page" .>: error calling partial: partial "edit_this_page" not found
ERROR 2022/11/30 18:36:32 render of "page" failed: "/drone/src/docs/themes/gitea/layouts/doc/single.html:14:8": execute of template failed: template: doc/single.html:14:8: executing "doc/single.html" at <partial "edit_this_page" .>: error calling partial: partial "edit_this_page" not found
```

**Comment by lunny** [comment]
make L-G-T-M work

**Pull request merged**